### ISSUE TITLE
Display identical nodes only once in compact list in planner

### ISSUE THREAD

**Issue by vmarc** [vmarc/knooppuntnet#404]
Suggestion from @pyrog:

One _"logical"_ node sometimes consists of multiple _"physical"_ guideposts on the ground. These guideposts have for example a distance of 10 or 20 meters between them, but they represent the same _"logical"_ node.

Currently, in the route planner compact node list, all _"physical"_ nodes are shown.  It would be useful to show the _"logical"_ node only once.  For example, nodes in the same relation could be displayed only once, or nodes with the same name with a distance shorter than a given threshold could be merged into one node.
